spec/services/nytimes_service_spec.rb

```diff
@@ -9,16 +9,17 @@
 			end
 
 			expect(Article.first.title).to(
-				eq("California Imposes Water Restrictions, in First for State")
+				eq("California Drought Tests History of Endless Growth")
 			)
 			expect(Article.first.url).to(
-				eq("http://www.nytimes.com/2015/04/02/us/california-imposes-first-" +
-					 "ever-water-restrictions-to-deal-with-drought.html")
+				eq("http://www.nytimes.com/2015/04/05/us/california-drought-tests-history" +
+           "-of-endless-growth.html")
 			)
 			expect(Article.first.abstract).to(
-				eq("The move by Gov. Jerry Brown comes as California’s drought has " +
-					 "reached near-crisis proportions after a winter that brought " +
-					 "record-low snowfalls.")
+				eq("The state&#8217;s history as a frontier of prosperity and glamour faces " +
```

Comment:
Line is too long. [83/80]<br>Indent the first parameter one step more than the previous line.<br>Tab detected.